tree_node.py

```diff
@@ -0,0 +1,5 @@
+class Node:
+    def __init__(self, value=None, left=None, right=None) -> None:
+        self.value = value
+        self.left = left
+        self.right = right
```

Comment:
Is `Node()`  valid way to initialise Node ? Can you tell me example where it'd be useful?

Reply:
Thought it will be useful for traversal to initialize dummynode. Not yet used. I will remove if not used in future also

Reply:
That can be done using `Node(None)` instead of having None as default value. Pick a preference and explain pros and cons

Reply:
There is already a node file used for Linked list which has prev and next node. Wanted to separate tree nodes from linked list node